src/Common/tests/System/Net/Capability.Sockets.cs

```diff
@@ -47,5 +47,10 @@ public static bool IPv4Support()
         {
             return Socket.OSSupportsIPv4;
         }
+
+        public static bool UnixDomainSocketsSupport()
+        {
+            return !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
In Common\src\System\Net\SocketProtocolSupportPal.Unix.cs SocketProtocolSupportPal is internal. If it's not easy to add a new internal method there, I'd rather remove UnixDomainSocketsSupport() from Capabilities then.